protocols/index.js

```diff
@@ -37,6 +37,7 @@ import palworld from './palworld.js'
 import quake1 from './quake1.js'
 import quake2 from './quake2.js'
 import quake3 from './quake3.js'
+import renegadex, { renegadexmaster } from './renegadex.js'
```

Comment:
Please split the master protocol query in another file.

Reply:
I would have thought protocols (as file) provide the logic for the single protocol. Thus as both classes are using the master-server, I kept it in the same file.

With possible changes (for instance with #608) the file structured could be scattered.